phpstan-baseline.neon

```diff
@@ -1,5 +1,10 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Call to an undefined method PHPStan\\\\Type\\\\Type\\:\\:getClassName\\(\\)\\.$#"
```

Comment:
You can't do this, it's not safe obviously.

Reply:
ah ok, i thought it was caused by removing the `instanceof` meaning that phpstan wasn't aware that the `isObject()->yes()` meant that function was available

Reply:
But it doesn't and shouldn't make that method available. Type is still Type.